cmd/osv-scanner/__snapshots__/main_test.snap

```diff
@@ -160,7 +160,7 @@ Loaded filter from: <rootdir>/fixtures/locks-many/osv-scanner.toml
           "informationUri": "https://github.com/google/osv-scanner",
           "name": "osv-scanner",
           "rules": [],
-          "version": "2.0.0-beta2"
+          "version": "2.0.0-beta3"
```

Comment:
nit: the CHANGELOG mentions rc1, but these tests still say beta3?

Reply:
Good catch updated now!